src/Movies/AddMovieButton.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { ReactElement } from 'react';
+import { Button } from '../components';
+
+export function AddMovieButton(): ReactElement {
+	const style: React.CSSProperties = {
+		backgroundColor: 'rgba(96, 96, 96, 0.68)',
+		textTransform: 'uppercase',
+		fontSize: '1.2rem',
+		fontWeight: '600'
+	}
+	return <Button style={style}>+ Add Movie</Button>
```

Comment:
Also usually each component should have its own folder (with all related files like styles, tests, storybook stories and so on)

Reply:
Done